chipsec/utilcmd/tpm_cmd.py

```diff
@@ -11,27 +12,65 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
+#You should have received a copy of the GNU General Public License
+#along with this program; if not, write to the Free Software
+#Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+#Contact information:
+#chipsec@intel.com
+#
 
 from chipsec.command import BaseCommand
 from chipsec.hal import tpm_eventlog
+from chipsec.hal import tpm
 
 class TPMCommand(BaseCommand):
-    """
-    >>> chipsec_util tpm parse_log <file>
-
-    Examples:
-
-    >>> chipsec_util tpm parse_log binary_bios_measurements
-    """
-    def requires_driver(self):
-        return False
+        """
+        >>> chipsec_util tpm parse_log <file>
+        >>> chipsec_util tpm state <locality>
+        >>> chipsec_util tpm command <commandName> <locality> <command_parameters>
 
-    def run(self):
-        if len(self.argv) < 4:
-            print TPMCommand.__doc__
-            return
-        log = open(self.argv[3])
-        tpm_eventlog.parse(log)
+        locality: 0 | 1 | 2 | 3 | 4
+        commands - parameters:
+        pccrread - pcr number ( 0 - 23 )
+        nvread - Index, Offset, Size
+        startup - startup type ( 1 - 3 )
+        continueselftest
+        getcap - Capabilities Area, Size of Sub-capabilities, Sub-capabilities
+        forceclear
+        
+        Examples:
+                        
+        >>> chipsec_util tpm parse_log binary_bios_measurements
+        >>> chipsec_util tpm state 0
+        >>> chipsec_util tpm command pcrread 0 17
+        >>> chipsec_util tpm command continueselftest 0
+        """
+        def requires_driver(self):
+            return False
 
+        def run(self):
```

Comment:
Need to create an instance of the TPM HAL class.  See the SPI command as an example.

_tpm = TPM(self.cs)